### DISCUSSION TITLE
can anyone help me with this 🙁

### DISCUSSION THREAD

**Discussion by PetaPat** [victornpb/undiscord#572] [1. Questions & Answers]
<img width="1427" alt="Screenshot 2023-12-28 at 3 21 43 AM" src="https://github.com/victornpb/undiscord/assets/154003247/388401d0-78ee-457c-a70c-992f56d607eb">
 

Idk what this means

**Comment by PetaPat** [comment]
And it doesn't work even if I put channel ID it coms up with the same message

**Comment by KuryloDev** [comment]
You can try to increase your search delay in "Advanced settings". It can happen pretty quickly that you get an empty list of messages during search; having the search delay to 6000 ms for example should completely negate that from happening. It will take quite a good chunk longer however.
